src/commands/generateSdk.ts

```diff
@@ -23,9 +23,61 @@ import { mapDbAstToSdkGeneratorAst } from "../generateSdk/utils/mapDbAstToFullAs
 import { generateSdk } from "../generateSdk/sdkGeneratorHandler.js";
 import { SdkGeneratorResponse } from "../models/sdkGeneratorResponse.js";
 import inquirer, { Answers } from "inquirer";
-import { GenezioSdkOptions } from "../models/commandOptions.js";
+import { GenezioSdkOptions, SourceType } from "../models/commandOptions.js";
+import { sdkGeneratorApiHandler } from "../generateSdk/generateSdkApi.js";
 
 export async function generateSdkCommand(projectName: string, options: GenezioSdkOptions) {
+    switch (options.source) {
+        case SourceType.local:
+            await generateLocalSdkCommand(options);
+            break;
+        case SourceType.remote:
+            await generateRemoteSdkCommand(projectName, options);
+            break;
+        default:
+            log.error("Invalid source type.");
+            exit(1);
+    }
+}
+
+export async function generateLocalSdkCommand(options: GenezioSdkOptions) {
+    let configuration: YamlProjectConfiguration = await YamlProjectConfiguration.create({
+        name: "test",
```

Comment:
Is this `test` name intended?

Reply:
Yea, it's kind of a hack because I am forced to create a `YamlProjectConfiguration` object and I have to provide a name.